cmd/rest/main.go

```diff
@@ -19,11 +19,11 @@ func main() {
 	}
 }
 
-// Run will bind our layers all together
+// Run will bind our layers all together.
 func Run() error {
 	config, err := cfg.GetConfig()
 	if err != nil {
-		return fmt.Errorf("failed to get config from .env: %+v", err)
+		return fmt.Errorf("failed to get config from .env: %+w", err)
```

Comment:
At this place, you don't know anything about the file name or about any file at all. This is the responsibility of Config and it's encapsulated from the Run level. Being acknowledged about the `.env` file at this point makes your code less abstracted and more coupled.

Reply:
fix

Reply:
You don't know about the file at this place.

Reply:
At last commit i fix it.